S32K/src/canfd.cpp

```diff
@@ -0,0 +1,904 @@
+/*
```

Comment:
So, the reason I had a folder that contained the cpp and the includes in my example change is we want to allow for git subtrees to be pulled that exclude all the extraneous test code. Can you introduce something like "module" to allow for this? So the top level tree would be:

```
module 
    |
    + canfd.cpp
    + libuavcan/media/S32K/canfd.hpp
test
    |
    + CMakeLists.txt
...
```

You also need to update CMakeLists.txt with the correct path the .cpp and include

Reply:
Implemented at https://github.com/noxuz/platform_specific_components/commit/408beca7fd8d688b37838983852aa706fed138ea